app/package.json

```diff
@@ -27,6 +27,7 @@
     "queue": "4.4.0",
     "semver": "5.4.1",
     "shell-env": "0.3.0",
+    "util.promisify": "false1.0.0",
```

Comment:
this is built in: https://nodejs.org/api/util.html#util_util_promisify_original

Reply:
I forgot to remove it from this file.
I replaced `util.promisify` by `pify`.
Promisify built-in is only available from node v8.x and we are using v7.9.0 (with Electron v1.7.8).

Reply:
👍  No probs